.github/workflows/native-binary-build-launcher.yml

```diff
@@ -0,0 +1,214 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# ----------------------
+#
+#  This workflow builds the native Windows launchers for the IDE and platform.
+#
+#  The result of the build are files 'launcher-external-sources.zip'
+#  and 'launcher-external-binaries.zip' which can be downloaded from
+#  the GitHub Actions UI and prepared for release to be used by the Ant build
+#  scripts for the NetBeans distribution.
+#
+# ----------------------
+
+
+name: NetBeans Native Launcher
+
+
+on:
+  push:
+#    branches:
+#      - 'master'
+#      - 'delivery'
+#      - 'release*'
```

Comment:
Not sure whether to uncomment or remove.  It makes developing a pain.  Needed to comment out to get the workflow to trigger on my own repo while working on it.  Not sure if we can limit the push branch filter to only upstream and not forks?  Otherwise, think we can lose the filters on all the native workflows.

Reply:
would workflow dispatch work for your usecase as trigger? I think you can even dispatch from gh CLI these days.

We have committers using branches on the main repo still which is the original reason why I added those. (see top of https://github.com/apache/netbeans/actions for a recent example)

Reply:
Workflow dispatch requires the workflow to be in the default branch.  I realise the reason for having the branch filter on the main workflow.  I think having branch _or_ path filter on push should be enough.

Reply:
I've removed the commented code for now.  We can add back if paths prove to not be enough in practice.

Reply:
ok, just be aware that this would be the only workflow without branch filter now.